src/views/AccessControl.tsx

```diff
@@ -127,26 +129,30 @@ export const AccessControl = () => {
     const saveKey = 'saving';
     useEffect(() => {
         if (savedRule.loading) {
-            message.loading({ content: 'Saving...', key: saveKey, duration: 0, style: styleNotification });
+            message.loading({ content: 'Saving...', key: saveKey, duration: 0, style: styleNotification })
         } else if (savedRule.success) {
-            message.success({ content: 'Rule saved with success!', key: saveKey, duration: 2, style: styleNotification });
-            dispatch(ruleActions.setSetupNewRuleVisible(false));
-            dispatch(ruleActions.setSavedRule({ ...savedRule, success: false }));
+            message.success({ content: 'Rule has been successfully updated.', key: saveKey, duration: 2, style: styleNotification });
+            dispatch(ruleActions.setSetupNewRuleVisible(false))
+            dispatch(ruleActions.setSavedRule({ ...savedRule, success: false }))
+            dispatch(ruleActions.resetSavedRule(null))
         } else if (savedRule.error) {
-            message.error({ content: 'Error! Something wrong to create key.', key: saveKey, duration: 2, style: styleNotification  });
-            dispatch(ruleActions.setSavedRule({ ...savedRule, error: null }));
+            message.error({ content: 'Failed to update rule. You might not have enough permissions.', key: saveKey, duration: 2, style: styleNotification  });
+            dispatch(ruleActions.setSavedRule({ ...savedRule, error: null }))
+            dispatch(ruleActions.resetSavedRule(null))
         }
     }, [savedRule])
 
     const deleteKey = 'deleting';
     useEffect(() => {
         const style = { marginTop: 85 }
         if (deletedRule.loading) {
-            message.loading({ content: 'Deleting...', key: deleteKey, style });
+            message.loading({ content: 'Deleting...', key: deleteKey, style })
         } else if (deletedRule.success) {
-            message.success({ content: 'Rule deleted with success!', key: deleteKey, duration: 2, style });
+            message.success({ content: 'Rule has been successfully disabled.', key: deleteKey, duration: 2, style })
+            dispatch(ruleActions.resetDeletedRule(null))
         } else if (deletedRule.error) {
-            message.error({ content: 'Error! Something wrong to delete rule.', key: deleteKey, duration: 2, style  });
+            message.error({ content: 'Failed to remove rule. You might not have enough permissions.', key: deleteKey, duration: 2, style  })
```

Comment:
Codacy has a fix for the issue: [Replace `······message.error({·content:·'Failed·to·remove·rule.·You·might·not·have·enough·permissions.',·key:·deleteKey,·duration:·2,·style··})` with `message.error({⏎········content:⏎··········"Failed·to·remove·rule.·You·might·not·have·enough·permissions.",⏎········key:·deleteKey,⏎········duration:·2,⏎········style,`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
      message.error({
```